packages/components/pie-card-container/src/index.ts

```diff
@@ -40,12 +40,20 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Boolean })
     public isDraggable = false;
 
+    @property({ type: String, reflect: true })
+    public padding!: PaddingValues;
+
+    @property({ type: String, reflect: true })
+    public paddingX!: PaddingValues;
+
```

Comment:
imo, these props should be marked as optional

```suggestion
    @property({ type: String, reflect: true })
    public padding?: PaddingValues;

    @property({ type: String, reflect: true })
    public paddingX?: PaddingValues;

```

the ! symbol or _definite assignment assertion operator_ indicates that the value must be provided beforehand by users and it isn't the case for padding.

Reply:
Good catch!